crates/context_aware_config/src/api/context/types.rs

```diff
@@ -81,6 +83,14 @@ pub struct PriorityRecomputeResponse {
     pub new_priority: i32,
 }
 
+#[derive(Serialize)]
+pub struct WeightRecomputeResponse {
+    pub id: String,
+    pub condition: Condition,
+    pub old_weight: BigDecimal,
```

Comment:
i think we should not send BigDecimal in the api response, we should convert it to String and send, as Javascript breaks on very large numbers and it rounds them off